docs/repositories-configure/integrations/github-integration.md

```diff
@@ -42,17 +42,29 @@ Adds a report to your pull requests showing whether your pull requests and cover
 
 ![Pull request status check on GitHub](images/github-integration-pr-status.png)
 
-### Annotations
+### Issue annotations
 
 Adds annotations on the lines of the pull request where Codacy finds new issues. Codacy maps the severity of the issues reported by the tools to the severity levels of the annotations. To enable this option, you must enable **Status checks** first.
 
-![Pull request annotation on GitHub](images/github-integration-pr-annotation.png)
+![Issue annotation on GitHub](images/github-integration-pr-annotation.png)
 
-### Summary
+### Issue summaries
 
 Shows an overall view of the changes in the pull request, including new issues and metrics such as complexity and duplication. To enable this option, you must enable **Status checks** first.
 
-![Pull request summary on GitHub](images/github-integration-pr-summary.png)
+![Issue summary on GitHub](images/github-integration-pr-summary.png)
+
+### Coverage summaries
+
+Adds a pull request comment showing an overall view of the coverage metrics for the pull request, including details about the data that Codacy used to calculate the coverage variation and diff coverage metrics.
```

Comment:
This is more a question than feedback, as I'm quite unsure: would it make sense to use the present tense and say `...data that Codacy uses to calculate...` here?

Reply:
The coverage summary includes information about the number of coverable and covered lines that Codacy received in the report that was used to calculate the final metrics. I believe that using the past tense makes more sense here because it helps to clarify the process that Codacy uses to calculate the final metrics: the final metrics are a result of the data that the user uploaded.